README.md

```diff
@@ -26,6 +36,53 @@ The MWI creates a score for each zip code between 0 and 100, such that:
+
+3. Unzip "hse-mwi-main.zip".
+
+4. In the unzipped folder, open "app.R" in RStudio. This should open RStudio and the "app.R" script in the top left hand corner of the application.
+
+5. In the console window, which is in the bottom left hand corner, enter the following line and answer "yes" to all prompts in the console as you install these packages:
+   * install.packages('readxl', 'writexl', 'htmltools', 'shiny', 'tigris', 'leaflet', 'RColorBrewer', 'sf', 'plotly', 'ggbeeswarm', 'shinyWidgets', 'sass', 'shinycssloaders', 'shinyBS')
+
+6. In "app.R", navigate to line 11, which should say "app_local <- FALSE". Change FALSE to TRUE.
+
+7. In the top right hand corner of the "app.R" window, you should see "Run App". Click the small downard arrow to the right of that and click "Run External". Then click "Run App".
+
+8. After a delay (this will be slow the first time, then quicker after that), the Mental Wellness Index Tool should open in your browser. Click on the "Create Your Own MWI" tab and follow the remaining steps to create your own MWI.
+
+9. If you are only adjusting weights for included data, skip the next step.
+
+10. Put each of your datasets in a CSV (comma separated value) format, with one column corresponding to the geographical ID of the data, a column corresponding to the numerator of the data, and another column corresponding to the denominator (if needed).
+   * Accepted geographical ID types are always numeric and include the following:
+      * ZCTA (example: 35406)
+      * County (example: 01001)
+      * ZIP Code (example: 35051)
+      * Census Tract (example: 01001020100)
+   * If a denominator column is provided, the final input to the MWI will be the numerator divided by the denominator, multiplied by the scaling number (specified in the metadata file, see next step).
+   * Numerators and denominators must be numeric columns.
+   * Missing data should have cells left blank.
+   * If race stratified, there should be two columns: one ending in '_pop' corresponding to the overall population measure, and one ending in '_black' corresponding to the black populations measure. In the Metadata.xlsx file edit, that row's 'Preprocessed' column should be set to TRUE.
+
+11. Download Metadata.xlsx with the button below. If adding custom data, add a row and fill in information for each measure you want to add to the Mental Wellness Index. Descriptions for each column can be found in the 'Column Descriptions' sheet of the Metadata.xlsx. Note that all column names, with the exception of 'denominator', must be filled out.
+   * If you have multiple measures in one file, add a row for each measure and its qualities, but specify the same file name.
+   * If you would not like to include a measure in your MWI, either delete the measure row or set its weight to 0.
```

Comment:
If you would like to remove an existing measure in your MWI, either delete the measure row or set it weight to 0.

Reply:
Fixed!